bool LED1
bool LED2
bool LED3
int8 batt_percent
